src/embed.ts

```diff
@@ -352,6 +352,18 @@ async function _embed(
     vega.timeFormatLocale(opts.timeFormatLocale);
   }
 
+  // Set custom expression functions
+  if (opts.expressionFunctions) {
+    for (const name in opts.expressionFunctions) {
+      const expressionFunction = opts.expressionFunctions[name];
+      if ('fn' in expressionFunction && 'visitor' in expressionFunction) {
```

Comment:
This won't work correctly if `visitor` is optional.  

use `isObject(expressionFunction)` instead ? 

(you can import `isObject` from vega, for consistency with other isString, etc.)